### PR TITLE
PE-91 | Implemented EureQa Logo and Edited Onboarding Screens

### PR DESCRIPTION
**Pull request by ayeejane**
What I Did: 

- Added the EureQa logo. 
- Edited the onboarding screens. 
- Removed unused images. 

How to Test: 

- Reinstall the app from your emulator or physical phone. 
- Go through the onboarding screens. Read the text as well.

### CODE REVIEW

File: lib/screens/onboarding_screen.dart [hunk 1]
```diff
@@ -11,66 +11,94 @@ class _OnboardingState extends State<Onboarding> {
...
         body:
-            'Asking and answering questions has never been so easy and rewarding \n You are ready to join! \n\n If you still have questions, visit our FAQ section',
-        decoration: const PageDecoration(
-          pageColor: Color(0xff6DB8FF),
+            'EureQa gives you an option of posting questions with an incentive.\n'
+            'Need an answer ASAP? \n\nTry adding incentives to your question. Paid questions get answered a lot faster.',
+        image: Image.asset(
+          'assets/images/incentives.png',
+          fit: BoxFit.fitWidth,
         ),
```
Comment (MaxLian11) [inline lib/screens/onboarding_screen.dart]: This slide is not necessary at this point as paying system has not been integrated yet. Consider to comment it out for now

**Review by MaxLian11 — APPROVED** [review]
Looks really good

**Review by tonynguyen98 — CHANGES_REQUESTED** [review]
Please remove the white background from the images, and can you possibly make our login screen also have the gradient as well?

**Review by dominikyz — APPROVED** [review]
Looks outstanding! Awesome job!

**Review by tonynguyen98 — APPROVED** [review]
Great work! I seems you've figured out a way to remove one of the screens.

**Pull request merged**